packages/app/package.json

```diff
@@ -14,9 +14,17 @@
     "/templates"
   ],
   "exports": {
-    "./plugins/extensions/*": {
```

Comment:
the new extensions plugins need to import Interfaces that are defined in `app`, those plugins will have `@shopify/app` as a peerDependency.
Other possibility would be to move all extensions Interfaces to cli-kit, but that didn't seem right, let me know if you need more clarification, we can pair if needed :)

Reply:
Just eliminate the possibility of having two copies of `@shopify/app` 
Since this plugin types are just for internal use, we can iterate on this if we detect friction in the process, to find the best way of extending not only `cli-kit` with plugins but also any other packages (maybe in the future we could have plugin hooks for themes!)